projects/storefrontapp/src/assets/i18n/register.en.json

```diff
@@ -20,14 +19,15 @@
     "firstName": "First Name",
     "lastName": "Last Name",
     "emailAddress": "Email Address",
-    "password": "Password"
+    "password": "Password",
+    "confirmPassword": "Confirm Password"
   },
   "validations": {
     "passwordMinRequirements": "Password must be six characters minimum, with one uppercase letter, one number, one symbol",
     "bothPasswordMustMatch": "Both password must match"
   },
-  "conditions": {
-    "useMyPersonalDataToReceiveEmailNewslettersFor": "Use my personal data to receive e-mail newsletters for marketing campaigns. To change your settings, go to Consent Management in My Account.",
-    "iAmConfirmingThatIHaveReadAndAgreedWith": "I am confirming that I have read and agreed with the"
+  "consents": {
```

Comment:
For simplicity of the structure (not to introduce new key), let's move it to `labels`:
- `emailMarketingConsent`
- `confirmThatRead`

Reply:
I separated it because these are descriptions of checkboxes and not individual labels